dev-resources/logback.xml

```diff
@@ -5,11 +5,11 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="error"/>
+    <logger name="puppetlabs" level="trace"/>
```

Comment:
Yeah, the behaviors of the two logging middleware log at trace and/or debug https://github.com/puppetlabs/ring-middleware/pull/28/files#diff-01da1d044c8bc8e6c87bddbbf3793a02R121